api/src/main/java/com/nicknathanjustin/streamercontracts/donations/OpenDonationDto.java

```diff
@@ -0,0 +1,30 @@
+package com.nicknathanjustin.streamercontracts.donations;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class OpenDonationDto {
```

Comment:
Hmmm I think we need to be careful about the vocabulary we use here. This object is really modeling a contract DTO...streamer name and description both come from the contracts table, not the donations table. 

I think what we really want to model here is "open contracts". The donations table is just a mechanism for supporting many donations on the same contract.